src/todolist/settings.py

```
@@ -18,7 +18,8 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.environ.get("SECRET_KEY", "dev-insecure-secret-change-me")
```

The SECRET_KEY is read from the environment, but a fallback value is provided. The requirements specify that there must be no hardcoded fallback that masks the absence of the Secret; absence should cause a startup error. Remove the fallback value so that the app fails to start if SECRET_KEY is not set (see checklist item 3.1).